tests/foreman/cli/test_ansible.py

```diff
@@ -561,3 +566,107 @@ def test_positive_install_ansible_collection(
         assert result['success'] == '1'
         collection_path = client.execute('ls ~/ansible_collections').stdout
         assert 'oasis_roles' in collection_path
+
+    @pytest.mark.tier3
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    @pytest.mark.parametrize('auth_type', ['admin', 'non-admin'])
+    def test_positive_ansible_variables_imported_with_roles(
+        self, request, auth_type, target_sat, module_org, module_ak_with_cv, rhel_contenthost
+    ):
+        """Verify that when Ansible roles are imported, their variables are imported simultaneously
```

Comment:
Should we mention here that the variables should be placed in the `/vars` directory? If they are in the `/defaults` directory, they will be imported successfully, which is not the case here.

Reply:
@nofaralfasi I think in both the case variables should be imported, but I think it'd be better to skip import from `vars/` directory as we've bug for importing from `vars/` directory currently, and to cover both I will add variables in both the directories later when the issue is fixed, until then we can import only from defaults dir. What do you think about that?

Reply:
I thought this test was intended to verify SAT-28198. If that’s not the case, I agree we should leave it as is, but perhaps we should remove the `:verifies: SAT-28198` annotation below.

Reply:
I agree with @nofaralfasi here, the test is different from bug : `SAT-28198` .

Reply:
Here as it currently covering defaults dir, but when the mentioned issue is fixed, then it'll start testing vars/ dir, which would cover the verification for SAT-28198, wouldn't that make sense?

Reply:
I'd suggest either splitting this into two separate tests or, at the very least, removing the `:verifies: SAT-28198` from the description. As it stands, it gives the impression that the functionality is working correctly when it actually isn't.

Reply:
no, it doesn't gives any such impression, we've many such bugs/issues in robottelo where we're handling it such a way.
Additionally, when mentioned issue will be fixed and is_open condition will fail, it'll start cover that issue and will cover testing from vars/ directory, but until then the defaults/ dir will be used instead